apps/s_client.c

```diff
@@ -3553,4 +3556,70 @@ static char *base64encode (const void *buf, size_t len)
     return out;
 }
 
+/*
+ * Host dNS Name verifier: used for checking that the hostname is in dNS format 
+ * before setting it as SNI
+ */
+static int is_dNS_name(const char *host)
+{
+    const size_t MAX_LABEL_LENGTH = 63;
+
```

Comment:
Delete the blank line here, keeping all the declarations together.

Reply:
Done